pwndbg/integration/binja.py

```diff
@@ -189,17 +192,11 @@ def can_connect() -> bool:
 
 
 def l2r(addr: int) -> int:
-    exe = pwndbg.aglib.elf.exe()
```

Comment:
why removing?

Reply:
The call to `.exe()` used to be there because I used it to get the exe base, but since the code uses `pwndbg.aglib.proc.binary_base_addr` now there's no point in the extra function call.